### PR TITLE
Ingest real time data into databse

### PR DESCRIPTION
**Pull request by Rebootony**
## step1
Read data from chargepoint APIs

## step2
Save data to Postgres database

## step3
Set timers.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,10 +2,11 @@
 terraform.tfstate
 terraform.tfstate.backup
 
-# Byte-compiled / optimized / DLL files
+# Byte-compiled / optimized / DLL files / Django migrations
 __pycache__/
 *.py[cod]
 *$py.class
+migrations
```
Comment (jongoncalves) [inline .gitignore]: We shouldn't ignore migration files from Django. This is how the databases are created/updated.

File: app/devine/chargepoint/chargepoint_api.py [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+           ['google', 'google_plymouth_sgID']]
+    retval = []
+    for site in sites:
+        retval.append(getData(site))
+        print('finish #')
+    # print(retval)
+    print('Total Time: ', time.time()-start)
+    return retval
+
+
```
Comment (jongoncalves) [inline app/devine/chargepoint/chargepoint_api.py:270]: Need consistent naming conventions throughout. You can check out the PEP 8 style guide, which we generally follow. But at a glance, pascal case the class name and snake case the function and variable names.

File: app/devine/chargepoint/config.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# slac_api -> slac_B53
+# slac-gismo_api -> slac_GISMO
+
+chargepoint_api_keys = {
+    'slac_api_username':'92d0023b4f07baa1857381a51de78772546674ef2b8391416000751',
+    'slac_api_password':'c2f058fc77298198f5bb3089cb354053',
+    'google_api_username': 'a0a3b3a5db088eef682a0252745dc5cf5f776a22a3ac71601661474',
+    'google_api_password':'7c0d2a84d442598fe5ea2db2cab71e97',
+    'slac-gismo_api_username': '7f74b2f7226e5cf50d59e84cc628649f5dadd5a5234f51571673509',
+    'slac-gismo_api_password': 'eb07d5826fb96b82efb9c130d26cafa6'
```
Comment (jongoncalves) [inline app/devine/chargepoint/config.py]: We need to remove passwords from github. We could pull this from env vars

Comment (Rebootony) [reply app/devine/chargepoint/config.py:13]: Ok, I see. Could you please tell me how to pull the password from env vars? I'm not familiar with that. Thank you!

**Review by jongoncalves — APPROVED** [review]

**Pull request merged**